components/stripe/stripe.app.js

```diff
@@ -51,26 +104,37 @@ module.exports = {
       type: "string",
       label: "Customer ID",
       description: "Example: `cus_Jz4ErxGo9t1agg`",
-      options: createOptionsMethod("customers", [
-        "id",
-        "name",
-      ]),
+      options: createOptionsMethod("customers", function ({
+        id, name, email,
+      }) {
+        return {
+          value: id,
+          label: [
+            name,
+            email,
+            id,
+          ].filter((v) => v).join(" | "),
```

Comment:
Addressing [this comment](https://github.com/PipedreamHQ/pipedream/pull/1859#issuecomment-974394222):
I changed the **Customer ID** option's label to
`customer_name (if available) | customer_email (if available) | customer_id`

For example:
- Customer with name, email, id:
`John Doe | email@example.com | cus_Jz4ErxGo9t1agg`
- Customer with without name:
`email@example.com | cus_Jz4ErxGo9t1agg`
- Customer without email:
`John Doe | cus_Jz4ErxGo9t1agg`
- Customer without name or email:
`cus_Jz4ErxGo9t1agg`

Is that what you had in mind @dannyroosevelt?

Reply:
yep -- I'm not sure if that's the most elegant way to display those data, but it's much improved from before. lgtm, thank you!